src/controllers/who.is.making.filing.controller.ts

```diff
@@ -12,9 +12,16 @@ import {
   OVERSEAS_ENTITY_DUE_DILIGENCE_WITH_PARAMS_URL
 } from "../config";
 import { getWhoIsFiling, postWhoIsFiling } from "../utils/who.is.making.filing";
+import { getApplicationData } from "../utils/application.data";
+import * as config from "../config";
```

Comment:
Lines 6-13 import specific variables from config rather than using the wildcard. Any more that are needed can be added there.

Reply:
I think my IDE (WebStorm) is adding these in automatically when I use a value prior to its importation. I'll fix this, and see if I can change the IDE to stop doing these auto-imports.